src/Scheduler.php

```diff
@@ -10,8 +10,15 @@
 class Scheduler {
 
 	public function setup() {
-		// Process the queue
-		add_action( 'shutdown', [ $this, 'process_queue' ], 999 );
+
+		/**
+		 * Make sure we aren't currently in a cron job or a rest request, so we don't create an infinite loop.
+		 */
+		if ( ! defined( 'DOING_CRON' ) && ! defined( 'REST_REQUEST' ) ) {
```

Comment:
if this doesn't get triggered during REST_REQUESTS, should the same be true for GRAPHQL_REQUEST(s)?

Reply:
or at least `GRAPHQL_HTTP_REQUEST`?

Reply:
Still seems like content created over REST _should_ trigger this at shutdown. 🤔

Reply:
@jasonbahl I'm pretty sure this was causing some infinite loops, where the callbacks were spawning processes to run themselves over and over again.

Reply:
The processors run in either a cron job or REST endpoint, so I wanted to prevent other processes from being spawned _from_ other processes.

Reply:
ya, I hear ya. disregard my comments.